src/test/scala/apps/separableConvolution2DRewrite.scala

```diff
@@ -157,6 +157,17 @@ class separableConvolution2DRewrite extends shine.test_util.Tests {
     ))
   }
 
+//  test("scanline to regRotSeq") {
+//    rewrite_steps(toTDSL(scanline)(weightsV)(weightsH), Seq(
+//      (repeatNTimes(2, topDown(lowering.reduceSeqUnroll)) `;`
+//        topDown(lowering.slideSeq(SlideSeq.Values, idE)) `;`
+//        BENF `;`
+//        topDown(algorithmic.slideSeqFusion) `;`
+//        topDown(lowering.mapSeq))
+//        -> toTDSL(regRotSeq)(weightsV)(weightsH)
+//    ))
+//  }
```

Comment:
I think we can remove this, the test is already in the rest of the code.

Reply:
Removed.